homeassistant/components/binary_sensor/ping.py

```diff
@@ -35,6 +35,9 @@
 PING_MATCHER = re.compile(
     r'(?P<min>\d+.\d+)\/(?P<avg>\d+.\d+)\/(?P<max>\d+.\d+)\/(?P<mdev>\d+.\d+)')
 
+PING_MATCHER_ALPINE = re.compile(
```

Comment:
Well it's really any ping that only returns 3 values, so there may be others that do this that I'm not aware of but this will support that. Calling it BUSYBOX seems just as arbitrary as ALPINE, but in the end doesn't matter, what is the preference?